.gitignore

```diff
@@ -6,3 +6,4 @@ uvicorn.egg-info/
 venv/
 htmlcov/
 site/
+/.idea/
```

Comment:
We reject any addition to the `.gitignore` that doesn't come from the repo build or specific tool in here.

Ref.: https://github.com/encode/uvicorn/pull/1001

```suggestion
```